content/posts/newsletter-007/index.md

```diff
@@ -529,6 +529,16 @@ February's full weekly devlogs: "This Week In Veloren...":
 
 ## Library & Tooling Updates
 
+### [Fix Your Time Step in Rust and GGEZ][fyt-ggez]
+
```

Comment:
Could you add an image from the article?

For example, https://joetsoi.github.io/fix-your-timestep-rust-ggez/interpolated.gif looks appropriate.

Reply:
👍 Sounds good. I'm at work for the next 8 hours but I might be able do it at lunch.

Reply:
Done (via https://gitpod.io, since I can't pull the repo to my work machine - very cool tool 😄)!